### ISSUE TITLE
Need release for Chef 14.4

### ISSUE THREAD

**Issue by lamont-granquist** [jarosser06/chef-zypper#14]
We've released the ohai, sudo, windows and build-essentials cookbooks to suppress the obnoxious Chef 15 deprecation warnings.  If you release #12 and #13 that'll get rid of:

```
Deprecated features used!
  Resource zypper_repo has been loaded from a cookbook. The resource zypper_repo is now included in Chef and will take precedence over the existing cookbook resource in the next major release of Chef (15.0, April 2019). You may be able to remove this cookbook dependency from your runlist if you do not use other recipes/resources/libraries from the cookbook. Alternatively there may be a newer version of this cookbook without the zypper_repo resource. at 1 location:
    - /opt/chef/embedded/lib/ruby/gems/2.5.0/gems/chef-14.4.56/lib/chef/log.rb:51:in `caller_location'
   See https://docs.chef.io/deprecations_map_collision.html for further details.
```

Chef 14.3.x users will still see that warning, but it'll be gone when they upgrade to Chef 14.4.x

**Comment by carlivar** [comment]
Yep we were just bit by this, too. The nginx cookbook depends on this one. Could you please release a new version @jarosser06 ?

**Comment by tophercullen** [comment]
@carlivar The needed changes have already been merged into master. However,  @jarosser06 has not pushed a new version of the cookbook (with those commits) it to the supermarket yet.

In the meantime, either use the cookbook from the git repo, or manually add the two lines.

**Comment by jarosser06** [comment]
I've tagged a new release in git but am having trouble getting stove to cooperate with me. I apologize for the delay but I just don't have the time to maintain this project anymore. I've put up the cookbook for adoption on supermarket.

In the meantime I'll try and get my environment working so I can upload the latest release to the Supermarket.

**Comment by lamont-granquist** [comment]
the zypper_repo / zypper_repository resource has actually been in core-chef for awhile now so there's no need to adopt the cookbook:

https://github.com/chef/chef/pull/5948

(that is only for zypper_repository, but the current code has a back-alias for zypper_repo)
